fem/fespace.hpp

```diff
@@ -656,6 +656,9 @@ class FiniteElementSpace
    /// Return update counter (see Mesh::sequence)
    long GetSequence() const { return sequence; }
 
+   /// Return whether or not the space is discontinuous (L2)
+   bool IsDGSpace() const { return dynamic_cast<const L2_FECollection*>(fec); }
```

Comment:
I'd suggest making the cast to bool explicit:
```suggestion
   bool IsDGSpace() const { return nullptr != dynamic_cast<const L2_FECollection*>(fec); }
```

Reply:
Good suggestion. Done.